### ISSUE TITLE
Several games missing from list

### ISSUE THREAD

**Issue by RaraCool** [BrianLima/UWPHook#83]
- **City Skylines**
- **Asphalt 9: Legends**
- Mindscanners

Apologies if this is not the correct place to ask about this. All these games are installed to my D: HDD, games in bold do not support installing to specific folders, and other games that also do not support this, like TES III and IV were recognised correctly.

**Comment by stevealexandre** [comment]
Hi, since a fix has been done which take a look of the exe file right now on the last release, can you try it?

**Comment by stevealexandre** [comment]
> I'm also having trouble adding Halo Wars 2.

Yes, already reported in #92. Go check there for the solution. The PR is created and wait the review to be merged.

**Comment by stevealexandre** [comment]
Same as previois, check other ticket.
PR is in review and will fix them.

**Comment by BrianLima** [comment]
Could you try the new version @burakcan, @Pinchcoder @RaraCool ?

https://github.com/BrianLima/UWPHook/releases/tag/2.13.0

**Comment by burakcan** [comment]
@BrianLima fh4 works with the latest version

**Comment by AlJoelson** [comment]
Sea of Thieves is also missing.

**Comment by stevealexandre** [comment]
Check issue #92 with releasz 2.4.1b https://github.com/stevealexandre/UWPHook/releases/tag/2.14.1b

**Comment by AlJoelson** [comment]
> Check issue #92 with releasz 2.4.1b https://github.com/stevealexandre/UWPHook/releases/tag/2.14.1b

Sea of Thieves doesn't work with this version.

**Comment by stevealexandre** [comment]
> > Check issue #92 with releasz 2.4.1b https://github.com/stevealexandre/UWPHook/releases/tag/2.14.1b
> 
> Sea of Thieves doesn't work with this version.

New release and tested with it. Try it : [2.15.0](https://github.com/stevealexandre/UWPHook/releases/tag/2.15.0)

**Comment by AlJoelson** [comment]
> > > Check issue #92 with releasz 2.4.1b https://github.com/stevealexandre/UWPHook/releases/tag/2.14.1b
> > 
> > 
> > Sea of Thieves doesn't work with this version.
> 
> New release and tested with it. Try it : [2.15.0](https://github.com/stevealexandre/UWPHook/releases/tag/2.15.0)

Nice! It works, I can add it to Steam now. Don't have the overlay but my mates can see that I'm in-game and know that I'm available to jump in and play with.

**Comment by stevealexandre** [comment]
> > > > Check issue #92 with releasz 2.4.1b https://github.com/stevealexandre/UWPHook/releases/tag/2.14.1b
> > > 
> > > 
> > > Sea of Thieves doesn't work with this version.
> > 
> > 
> > New release and tested with it. Try it : [2.15.0](https://github.com/stevealexandre/UWPHook/releases/tag/2.15.0)
> 
> Nice! It works, I can add it to Steam now. Don't have the overlay but my mates can see that I'm in-game and know that I'm available to jump in and play with.

Great.
It's the expected behavior. UWPHook act as the app running for steam and run the game for you so in reality the overlay is available on the UWPHook which is running in backgroup until your game is closed. (this is how the app can show you playing as)
Unfortunately, it's impossible to have it on your game because else, you could run your game directly from steam. (Thanks xbox gamepass implementation...)
